generated/nidaqmx/_install_daqmx.py

```diff
@@ -5,23 +5,30 @@
 import logging
 import os
 import pathlib
+import re
+import shutil
 import subprocess
 import sys
 import tempfile
 import traceback
 import urllib.request
+import zipfile
 from typing import Generator, List, Optional, Tuple
 
 import click
 
 if sys.platform.startswith("win"):
     import winreg
+elif sys.platform.startswith("linux"):
+    import distro
```

Comment:
Consider:

```suggestion
_IS_WINDOWS =  sys.platform.startswith("win")
_IS_LINUX = sys.platform.startswith("linux")

if _IS_WINDOWS :
    import winreg
elif _IS_LINUX:
    import distro
```

this is a shorter form for all `if` statements in the code below. AND, if there is a reason to change the logic for detection, using a private "const" like this makes it easy to only change it in one place and have all references (currently copies) in sync.

Reply:
I don't think this works with mypy.

https://mypy.readthedocs.io/en/stable/common_issues.html#python-version-and-system-platform-checks

> Note
>
> Mypy currently does not support more complex checks, and does not assign any special meaning when assigning a [sys.version_info](https://docs.python.org/3/library/sys.html#sys.version_info) or [sys.platform](https://docs.python.org/3/library/sys.html#sys.platform) check to a variable. This may change in future versions of mypy.

Reply:
Yeah, it's always fun when the cleaner, simpler code does not work with tooling.